src/css/custom.css

```diff
@@ -65,6 +65,16 @@
     @apply font-sans antialiased;
   }
 
+  table td:nth-child(1) {
+    overflow-wrap: normal;
```

Comment:
Is the out-of-the-box Docusaurus styling different between column 1 and columns 2..n? What's the reason to have different `overflow-wrap` settings?